docs/sources/tempo/configuration/_index.md

```diff
@@ -1,15 +1,16 @@
 ---
-title: Configuration
+title: Configure Tempo
```

Comment:
The main menu has:
```
- Set up Tempo
- Configure Tempo
- Tempo API
```
as top level items. Since we are in Tempo docs should we drop the word "Tempo" on these:
```
- Set up
- Configure
- API
```